### ISSUE TITLE
New workspaces have a lot of random, unreadable files

### ISSUE THREAD

**Issue by mccalluc** [hubmapconsortium/user_workspaces_server#56]
I've started a workspace, and there are a lot of files in in besides the notebook I created:
<img width="369" alt="Screen Shot 2022-06-28 at 4 36 11 PM" src="https://user-images.githubusercontent.com/730388/176281543-86911b9f-7e36-4a5c-8e38-4e67e15f7419.png">
If I try to open in the UI:
> File Load Error for 1656438429.919012.sh
> Invalid response: 404 Not Found

But the same files are readable from the shell:
```
mccalluc@1605958a36bd:~/43$ ls
1656438429.919012.sh   JupyterLabJob_50_config.py   JupyterLabJob_51_config.py   notebook.ipynb
1656445414.2997541.sh  JupyterLabJob_50_output.log  JupyterLabJob_51_output.log
1656445508.3097498.sh  JupyterLabJob_50_venv        JupyterLabJob_51_venv
mccalluc@1605958a36bd:~/43$ head 1656438429.919012.sh
#!/bin/bash
# Let's assume that the module/virtualenv set-up can be moved else-where
# TODO: Make this configurable
virtualenv -p python3.8 "JupyterLabJob_49_venv"
source "JupyterLabJob_49_venv/bin/activate"
pip install jupyterlab

# All we really need is this part

CONFIG_FILE="$(pwd)/JupyterLabJob_49_config.py"
```

- Should these be visible at all? They seem like a distraction.
- If they should be visible, they should be openable in the UI without a 404

**Comment by jpuerto-psc** [comment]
Consideration:
- Move these files/directories under a "jobs" directory to avoid confusion.

Investigate:
- Non-Jupyter Notebooks cannot be read from JupyterLab session (throw 404). Issue appears to be related to query params, as the URL w/o those query params returns file metadata.

**Comment by jpuerto-psc** [comment]
Solution:
- Hiding config/scripts for jobs under a dot directory so that they're not visible from the JupyterLab UI. 

Investigate:
- Still need to investigate the 404 errors since this would cause issues with any scripts/files that are created through the API that aren't Jupyter Notebooks.

**Comment by jpuerto-psc** [comment]
Solution for un-readable files:

TLDR: Keyword conflict between dependency and JupyterLab

django-rest-framework relies on `format` in the query_params to "intelligently" determine what type of output to generate. JupyterLab **also** relies on this same parameter. This was causing conflicts as django-rest-framework was not routing calls appropriately. Turning off this setting `URL_FORMAT_OVERRIDE` resolved this issue.
